realtime_extension.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+import "gtfs-realtime.proto";
+
+option java_package = "de.mfdz";
+package transit_realtime;
+
+message MfdzTripDescriptorExtension {
+  optional string trip_url = 1;
+}
+
+extend TripDescriptor {
+  optional MfdzTripDescriptorExtension trip_descriptor = 1100;
```

Comment:
Shouldn't we use an extension id >= 9000 instead? (see https://github.com/google/transit/pull/178/)

Reply:
The `realtime-transit.proto` file in this project is not up-to-date with the upstream one so it doesn't have those modifications yet. If I update it, presumably the diff will become a lot messier as the Java files are also committed to this repo.